### PR TITLE
read me

### PR DESCRIPTION
**Pull request by ialberquilla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This repository contains our project for the Chainlink fall 2022 Hackathon 
+
+## Introduction
+The scope of this project is to create technical indicators for trading that can be consumed directly on-chain. Right now chainlink provides services as oracle providing prices to protocols, but in some cases for more complex trading strategies trading indicators are needed and can’t be done directly on-chain because would be extremely costly to do these calculations, this is why we created this project to provide technical indicators in a decentralized way. 
+
+In order to push these indicators on-chain, we wanted to do as most as decentralized and trust minimized way. We wanted to be able to have all the stack decentralized and allow anybody to track the value of the indicators on-chain, to let people trust in the system and ensure that if a rebalance had happened is because it has to happen and not have to trust in a specific centralized server. 
+
+Right now, this service supports the following indicators:
+
+- SMA: Single moving average
```
Comment (fabriziogianni7) [inline README.md:11]: simple moving average

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- EMA: Exponential moving average
+- BB: Bolliger bands
+- RSI: Relative Strength Index
+- ATR: Average true range
+
+
+## Architecture
+To create this architecture  in a decentralized way, we are using two chainlink services:
+External adapter: This adapter is responsible to calculate the indicator, it leverages the chainlink architecture to receive an on-chain call, calculate the indicator value, and put the response again on the chain. Again to use decentralized pricing, we use the price feed from chainlink, using a The graph subgraph to get the historical prices and perform the calculations over them.
```
Comment (fabriziogianni7) [inline README.md:20]: should mention also our node?

Comment (ialberquilla) [reply README.md:20]: Yes, maybe you can do it I don't know the details

Comment (fabriziogianni7) [reply README.md:20]: maybe we can say that we set up a real chainlink node and prepared the job for the adapter so any other node can use it

File: README.md (hunk 1 above)
Comment (ialberquilla) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (fabriziogianni7) [reply README.md:20]: ok I'll do it

**Review by fabriziogianni7 — APPROVED** [review]

**Pull request merged**